test/functional/viml/completion_spec.lua

```diff
@@ -717,3 +718,33 @@ describe('completion', function()
 
 end)
 
+describe('command line completion', function()
```

Comment:
This particular test should be added to `test/functional/ui/wildmode_spec.lua`. (wildmode/wildmenu is separate from insert-mode completion.)